framework/src/timesteppers/FunctionDT.C

```diff
@@ -80,27 +125,32 @@ FunctionDT::computeDT()
 {
   Real local_dt = 0;
 
-  if (_interpolate)
-    local_dt = _time_ipol->sample(_time);
-  else // Find where we are
+  if (_use_function)
+    local_dt = _function->value(_time, Point(0, 0, 0));
```

Comment:
Minor: Creating a temporary here may have a speed impact for applications that take many fast time steps.

Reply:
I am thinking of adding `_zero_point` to `ZeroInterface`. There is a few instances in the framework where we use it. And apps could benefit as well... But, I will do that in a different PR.